presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetFileReader.java

```diff
@@ -80,7 +68,9 @@ public ParquetColumnChunkPageReader readColumn(ColumnDescriptor columnDescriptor
     {
         checkArgument(currentBlockMetadata.getRowCount() > 0, "Row group having 0 rows");
 
-        ColumnChunkMetaData metadata = columnMetadata.get(columnDescriptor);
+        ColumnChunkMetaData metadata = getColumnChunkMetaData(columnDescriptor);
+        checkArgument(metadata != null, "Could not find column metadata in the parquet file");
```

Comment:
why not put this in `getColumnChunkMetaData` and then the method would never return null